src/vm/arm/stubs.cpp

```diff
@@ -366,6 +366,9 @@ void ValidateWriteBarriers()
 }
 #endif // _DEBUG
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Winvalid-token-paste"
```

Comment:
Do we use clang diagnostic push anywhere else?  If not can we clean up this file, or just add it globally?  This could leak all over the code base quickly.